content/docs/2.7/concepts/scaling-deployments.md

```diff
@@ -195,6 +192,21 @@ Starting from Kubernetes v1.18 the autoscaling API allows scaling behavior to be
 
 **Assumptions:** KEDA must be running on Kubernetes cluster v1.18+, in order to be able to benefit from this setting.
 
+---
+#### triggers
+```yaml
+  triggers:
+  # {list of triggers to activate scaling of the target resource}
```

Comment:
Should we add an example?

Reply:
I don't think it is needed.

Reply:
I didn't think so either because of the big shiny reference to all scalers below.

Reply:
Resolving this without adding an example, feel free to unresolve if you disagree.